helm_deploy/values-prod.yaml

```diff
@@ -50,5 +51,7 @@ generic-prometheus-alerts:
     - "Digital-Prison-Services-prod-prisoner_to_nomis_appointment_dlq"
     - "Digital-Prison-Services-prod-hmpps_prisoner_to_nomis_sentencing_queue"
     - "Digital-Prison-Services-prod-hmpps_prisoner_to_nomis_sentencing_dlq"
+    - "syscon-devs-prod-hmpps_prisoner_to_nomis_adjudication_dead_letter_queue"
```

Comment:
queues created in this namespace rather than hmpps-domain-events hence the different naming convention